package.json

```diff
@@ -5,10 +5,10 @@
     "test:unit": "jest",
     "test:unit:watch": "jest --watch",
     "lint": "vue-tsc --noEmit --declaration false && eslint --ext .js,.ts,.vue src",
-    "storybook": "SHOW_INTERACTIONS=true start-storybook -p 6006",
+    "storybook": "SHOW_INTERACTIONS=true storybook dev -p 6006",
     "build-vue": "rollup -c",
-    "build-storybook": "build-storybook",
-    "test-storybook": "test-storybook --url http://127.0.0.1:6006",
+    "build-storybook": "storybook build",
+    "test-storybook": "test-storybook --url http://0.0.0.0:6006",
```

Comment:
Is there a reason why you changed it back to 0.0.0.0? Sometimes the GH Runner has problems with this URL